README.md

```diff
@@ -113,6 +113,45 @@ There are two events that can be used as triggers for this action, each with dif
 1. [`pull_request_target`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target): This allows the action to be used in a fork-based workflow, where e.g. you want to accept pull requests in a public repository. In this case, the configuration from the main branch of your repository will be used for the check. This means that you need to have this configuration in the main branch for the action to run at all (e.g. it won't run within a PR that adds the action initially). Also if you change the configuration in a PR, the changes will not be reflected for the current PR – only subsequent ones after the changes are in the main branch.
 2. [`pull_request`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request): This configuration uses the latest configuration that is available in the current branch. It will only work if the branch is based in the repository itself. If this configuration is used and a pull request from a fork is opened, you'll encounter an error as the GitHub token environment parameter is not available. This option is viable if all contributors have write access to the repository.
 
+## Outputs
```

Comment:
@juninholiveira I've shortened the section a bit, I hope that's ok for you! Can you have a look if this looks complete to you?

https://github.com/amannn/action-semantic-pull-request/blob/cb5e1aeaea5c2c433ef5fe17ae40e104ee900aac/README.md

Reply:
It's great! Concise.